src/webgl.js

```diff
@@ -343,8 +345,8 @@ function init_conv1_2_program(gl) {
 
   const conv1_2_shader = `#version 300 es
 
-  precision highp float;
-  precision highp sampler2D;
+  precision mediump float;
+  precision mediump sampler2D;
```

Comment:
Oh neat, I didn't know you could set a precision on samplers like this. I think that could have a very significant impact on shader performance.